Bug: Dockerfile.train not building (python 3.6 vs 3.8 syntax error)

Building `Dockerfile.train` throws `SyntaxError` and errors out. Looks like a Python syntax issue where[ `:=` was introduced in python 3.8](https://docs.python.org/3/whatsnew/3.8.html#assignment-expressions)

## Steps to reproduce

### Step 1

clone `STT`

```
$ git clone https://github.com/coqui-ai/STT.git
$ cd STT
```

### Step 2

pull `kenlm`

```
STT$ git submodule update --init kenlm
Submodule 'kenlm' (https://github.com/kpu/kenlm) registered for path 'kenlm'
Cloning into '/home/ubuntu/foo/STT/kenlm'...
Submodule path 'kenlm': checked out 'bbf4fc511266c5d4515047055d7bdec659a6e158'
```
### Step 3

finally, run `docker build`. This is where the error is thrown

```
STT$ docker build -f Dockerfile.train -t stt-train-local:latest .
Sending build context to Docker daemon  100.7MB
Step 1/27 : FROM nvcr.io/nvidia/tensorflow:20.06-tf1-py3 AS kenlm-build
 ---> 61568efc3e0e
Step 2/27 : ENV DEBIAN_FRONTEND=noninteractive
 ---> Using cache
 ---> 887044fb322e
Step 3/27 : RUN apt-get update &&     apt-get install -y --no-install-recommends     build-essential cmake libboost-system-dev     libboost-thread-dev libboost-program-options-dev     libboost-test-dev libeigen3-dev zlib1g-dev     libbz2-dev liblzma-dev &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> f5b6fe39e51a
Step 4/27 : WORKDIR /code
 ---> Using cache
 ---> 6435f2612417
Step 5/27 : COPY kenlm /code/kenlm
 ---> Using cache
 ---> b80c97cee8e8
Step 6/27 : RUN cd /code/kenlm &&     mkdir -p build &&     cd build &&     cmake .. &&     make -j $(nproc) ||     ( echo "ERROR: Failed to build KenLM.";     echo "ERROR: Make sure you update the kenlm submodule on host before building this Dockerfile.";     echo "ERROR: $ cd STT; git submodule update --init kenlm";     exit 1; )
 ---> Using cache
 ---> debf2598444d
Step 7/27 : FROM nvcr.io/nvidia/tensorflow:20.06-tf1-py3
 ---> 61568efc3e0e
Step 8/27 : ENV DEBIAN_FRONTEND=noninteractive
 ---> Using cache
 ---> 887044fb322e
Step 9/27 : RUN apt-get update &&     apt-get install -y --no-install-recommends         git         wget         libopus0         libopusfile0         libsndfile1         sox         libsox-fmt-mp3 &&     rm -rf /var/lib/apt/lists/*
 ---> Using cache
 ---> 392d06bbd7a6
Step 10/27 : RUN pip3 install --upgrade pip wheel setuptools
 ---> Using cache
 ---> 5eae7b202dbf
Step 11/27 : WORKDIR /code
 ---> Using cache
 ---> 0fd27b2e33a1
Step 12/27 : COPY native_client /code/native_client
 ---> Using cache
 ---> 61bd9369290e
Step 13/27 : COPY .git /code/.git
 ---> e519f14e6ffa
Step 14/27 : COPY training/coqui_stt_training/VERSION /code/training/coqui_stt_training/VERSION
 ---> 6f620f1147b5
Step 15/27 : COPY training/coqui_stt_training/GRAPH_VERSION /code/training/coqui_stt_training/GRAPH_VERSION
 ---> 9f1f6ce0a262
Step 16/27 : RUN cd native_client/ctcdecode && make NUM_PROCESSES=$(nproc) bindings
 ---> Running in 09a24518a7f1
rm -rf dist temp_build coqui_stt_ctcdecoder.egg-info
rm -f swigwrapper_wrap.cpp swigwrapper.py first_party.a
mkdir -p bazel-out/ && \
	../bazel_workspace_status_cmd.sh > bazel-out/stable-status.txt && \
	../gen_workspace_status.sh > workspace_status.cc
++ git describe --long --tags
+ tf_git_rev=v1.3.0-9-g2bc61aba
+ echo 'STABLE_TF_GIT_VERSION v1.3.0-9-g2bc61aba'
++ dirname ../bazel_workspace_status_cmd.sh
+ pushd ..
++ git describe --long --tags
+ ds_git_rev=v1.3.0-9-g2bc61aba
+ echo 'STABLE_DS_GIT_VERSION v1.3.0-9-g2bc61aba'
++ cat ../training/coqui_stt_training/VERSION
+ ds_version=1.3.0
+ echo 'STABLE_DS_VERSION 1.3.0'
++ cat ../training/coqui_stt_training/GRAPH_VERSION
+ ds_graph_version=6
+ echo 'STABLE_DS_GRAPH_VERSION 6'
+ popd
++ grep STABLE_TF_GIT_VERSION bazel-out/stable-status.txt
++ cut '-d ' -f2
+ tf_git_version=v1.3.0-9-g2bc61aba
++ grep STABLE_DS_VERSION bazel-out/stable-status.txt
++ cut '-d ' -f2
+ ds_version=1.3.0
++ grep STABLE_DS_GIT_VERSION bazel-out/stable-status.txt
++ cut '-d ' -f2
+ ds_git_version=v1.3.0-9-g2bc61aba
++ grep STABLE_DS_GRAPH_VERSION bazel-out/stable-status.txt
++ cut '-d ' -f2
+ ds_graph_version=6
+ cat
mkdir -p /code/native_client/ds-swig
curl -sSL ""https://github.com/mozilla/DeepSpeech/releases/download/v0.9.3/ds-swig.linux.amd64.tar.gz"" | tar -C /code/native_client/ds-swig -zxf -
ln -s ds-swig /code/native_client/ds-swig/bin/swig
SWIG_LIB="/code/native_client/ds-swig/share/swig/4.1.0/" PATH="/code/native_client/ds-swig/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin:/opt/tensorrt/bin" swig -version

SWIG Version 4.1.0

Compiled with g++ [x86_64-pc-linux-gnu]

Configured options: +pcre

Please see http://www.swig.org for reporting bugs and further information
SWIG_LIB="/code/native_client/ds-swig/share/swig/4.1.0/" PATH="/code/native_client/ds-swig/bin:/usr/local/mpi/bin:/usr/local/nvidia/bin:/usr/local/cuda/bin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/local/ucx/bin:/opt/tensorrt/bin" swig -swiglib
/code/native_client/ds-swig/share/swig/4.1.0/
python -m pip install --quiet numpy wheel setuptools
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
DISTUTILS_USE_SDK=1 PATH=/code/native_client/ds-swig/bin::$PATH SWIG_LIB="/code/native_client/ds-swig/share/swig/4.1.0/" AS=as CC=gcc CXX=c++ LD=ld LIBEXE= CFLAGS="   " LDFLAGS="-Wl,--no-as-needed"   python ./setup.py build_ext --num_processes 36 
mkdir temp_build/temp_build/../kenlm/util
mkdir temp_build/temp_build/../kenlm/lm
mkdir temp_build/temp_build/../kenlm/util/double-conversion
mkdir temp_build/temp_build/third_party/openfst-1.6.7/src/lib
mkdir temp_build/temp_build/third_party/flashlight/flashlight/lib/common
mkdir temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder
mkdir temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/lm
mkdir temp_build/temp_build/third_party/flashlight/flashlight/lib/text/dictionary
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/parallel_read.cc -o temp_build/temp_build/../kenlm/util/parallel_read.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/integer_to_string.cc -o temp_build/temp_build/../kenlm/util/integer_to_string.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/float_to_string.cc -o temp_build/temp_build/../kenlm/util/float_to_string.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/string_piece.cc -o temp_build/temp_build/../kenlm/util/string_piece.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/mmap.cc -o temp_build/temp_build/../kenlm/util/mmap.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/murmur_hash.cc -o temp_build/temp_build/../kenlm/util/murmur_hash.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/lm_exception.cc -o temp_build/temp_build/../kenlm/lm/lm_exception.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/exception.cc -o temp_build/temp_build/../kenlm/util/exception.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/trie.cc -o temp_build/temp_build/../kenlm/lm/trie.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/ersatz_progress.cc -o temp_build/temp_build/../kenlm/util/ersatz_progress.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/file.cc -o temp_build/temp_build/../kenlm/util/file.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/read_compressed.cc -o temp_build/temp_build/../kenlm/util/read_compressed.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/spaces.cc -o temp_build/temp_build/../kenlm/util/spaces.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/file_piece.cc -o temp_build/temp_build/../kenlm/util/file_piece.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/sizes.cc -o temp_build/temp_build/../kenlm/lm/sizes.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/quantize.cc -o temp_build/temp_build/../kenlm/lm/quantize.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/cached-powers.cc -o temp_build/temp_build/../kenlm/util/double-conversion/cached-powers.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/pool.cc -o temp_build/temp_build/../kenlm/util/pool.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/search_trie.cc -o temp_build/temp_build/../kenlm/lm/search_trie.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/config.cc -o temp_build/temp_build/../kenlm/lm/config.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/strtod.cc -o temp_build/temp_build/../kenlm/util/double-conversion/strtod.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/trie_sort.cc -o temp_build/temp_build/../kenlm/lm/trie_sort.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/bignum.cc -o temp_build/temp_build/../kenlm/util/double-conversion/bignum.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/virtual_interface.cc -o temp_build/temp_build/../kenlm/lm/virtual_interface.oc++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/double-conversion.cc -o temp_build/temp_build/../kenlm/util/double-conversion/double-conversion.o

c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/read_arpa.cc -o temp_build/temp_build/../kenlm/lm/read_arpa.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/vocab.cc -o temp_build/temp_build/../kenlm/lm/vocab.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/search_hashed.cc -o temp_build/temp_build/../kenlm/lm/search_hashed.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/diy-fp.cc -o temp_build/temp_build/../kenlm/util/double-conversion/diy-fp.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/bit_packing.cc -o temp_build/temp_build/../kenlm/util/bit_packing.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/usage.cc -o temp_build/temp_build/../kenlm/util/usage.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/value_build.cc -o temp_build/temp_build/../kenlm/lm/value_build.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/model.cc -o temp_build/temp_build/../kenlm/lm/model.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/bhiksha.cc -o temp_build/temp_build/../kenlm/lm/bhiksha.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/lm/binary_format.cc -o temp_build/temp_build/../kenlm/lm/binary_format.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/scoped.cc -o temp_build/temp_build/../kenlm/util/scoped.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/fixed-dtoa.cc -o temp_build/temp_build/../kenlm/util/double-conversion/fixed-dtoa.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/fast-dtoa.cc -o temp_build/temp_build/../kenlm/util/double-conversion/fast-dtoa.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../kenlm/util/double-conversion/bignum-dtoa.cc -o temp_build/temp_build/../kenlm/util/double-conversion/bignum-dtoa.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/weight.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/weight.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/util.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/util.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/properties.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/properties.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/fst.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/fst.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/fst-types.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/fst-types.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/compat.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/compat.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/flags.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/flags.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/mapped-file.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/mapped-file.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/symbol-table.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/symbol-table.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/openfst-1.6.7/src/lib/symbol-table-ops.cc -o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/symbol-table-ops.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/common/String.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/common/String.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/common/System.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/common/System.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/decoder/LexiconDecoder.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/LexiconDecoder.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/decoder/LexiconFreeDecoder.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/LexiconFreeDecoder.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/decoder/lm/ConvLM.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/lm/ConvLM.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/decoder/lm/KenLM.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/lm/KenLM.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/decoder/lm/ZeroLM.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/lm/ZeroLM.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/decoder/Trie.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/Trie.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/decoder/Utils.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/Utils.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/dictionary/Dictionary.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/dictionary/Dictionary.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight third_party/flashlight/flashlight/lib/text/dictionary/Utils.cpp -o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/dictionary/Utils.o
ar rcs third_party.a temp_build/temp_build/../kenlm/util/murmur_hash.o temp_build/temp_build/../kenlm/util/spaces.o temp_build/temp_build/../kenlm/util/float_to_string.o temp_build/temp_build/../kenlm/util/double-conversion/diy-fp.o temp_build/temp_build/../kenlm/util/double-conversion/cached-powers.o temp_build/temp_build/../kenlm/util/double-conversion/fast-dtoa.o temp_build/temp_build/../kenlm/util/double-conversion/bignum-dtoa.o temp_build/temp_build/../kenlm/util/double-conversion/strtod.o temp_build/temp_build/../kenlm/lm/lm_exception.o temp_build/temp_build/../kenlm/util/double-conversion/fixed-dtoa.o temp_build/temp_build/../kenlm/util/ersatz_progress.o temp_build/temp_build/../kenlm/util/parallel_read.o temp_build/temp_build/../kenlm/lm/virtual_interface.o temp_build/temp_build/../kenlm/util/exception.o temp_build/temp_build/../kenlm/util/integer_to_string.o temp_build/temp_build/../kenlm/lm/config.o temp_build/temp_build/../kenlm/util/pool.o temp_build/temp_build/../kenlm/util/bit_packing.o temp_build/temp_build/../kenlm/util/string_piece.o temp_build/temp_build/../kenlm/util/scoped.o temp_build/temp_build/../kenlm/lm/trie.o temp_build/temp_build/../kenlm/util/read_compressed.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/Utils.o temp_build/temp_build/../kenlm/lm/bhiksha.o temp_build/temp_build/../kenlm/util/mmap.o temp_build/temp_build/../kenlm/util/double-conversion/double-conversion.o temp_build/temp_build/../kenlm/lm/quantize.o temp_build/temp_build/../kenlm/util/double-conversion/bignum.o temp_build/temp_build/../kenlm/lm/sizes.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/compat.o temp_build/temp_build/../kenlm/util/file_piece.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/properties.o temp_build/temp_build/../kenlm/lm/binary_format.o temp_build/temp_build/../kenlm/lm/read_arpa.o temp_build/temp_build/../kenlm/util/file.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/mapped-file.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/util.o temp_build/temp_build/../kenlm/lm/value_build.o temp_build/temp_build/third_party/flashlight/flashlight/lib/common/String.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/lm/ZeroLM.o temp_build/temp_build/../kenlm/util/usage.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/weight.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/Trie.o temp_build/temp_build/third_party/flashlight/flashlight/lib/common/System.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/lm/KenLM.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/dictionary/Dictionary.o temp_build/temp_build/../kenlm/lm/vocab.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/dictionary/Utils.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/fst.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/flags.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/lm/ConvLM.o temp_build/temp_build/../kenlm/lm/search_hashed.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/LexiconFreeDecoder.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/symbol-table-ops.o temp_build/temp_build/third_party/flashlight/flashlight/lib/text/decoder/LexiconDecoder.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/symbol-table.o temp_build/temp_build/../kenlm/lm/trie_sort.o temp_build/temp_build/../kenlm/lm/model.o temp_build/temp_build/../kenlm/lm/search_trie.o temp_build/temp_build/third_party/openfst-1.6.7/src/lib/fst-types.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ctc_beam_search_decoder.cpp -o temp_build/temp_build/ctc_beam_search_decoder.oscorer.cpp:32:30: warning: multi-character character constant [-Wmultichar]
 static const int32_t MAGIC = 'TRIE';
                              ^~~~~~
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight scorer.cpp -o temp_build/temp_build/scorer.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight path_trie.cpp -o temp_build/temp_build/path_trie.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight decoder_utils.cpp -o temp_build/temp_build/decoder_utils.o

c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight workspace_status.cc -o temp_build/temp_build/workspace_status.o
c++ -c     -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight ../alphabet.cc -o temp_build/temp_build/../alphabet.o
ar rcs first_party.a temp_build/temp_build/workspace_status.o temp_build/temp_build/../alphabet.o temp_build/temp_build/path_trie.o temp_build/temp_build/decoder_utils.o temp_build/temp_build/ctc_beam_search_decoder.o temp_build/temp_build/scorer.o
running build_ext
building 'coqui_stt_ctcdecoder._swigwrapper' extension
swigging swigwrapper.i to swigwrapper_wrap.cpp
swig -python -c++ -extranative -o swigwrapper_wrap.cpp swigwrapper.i
../alphabet.h:20: Warning 362: operator= ignored
scorer.h:28: Warning 401: Nothing known about base class 'fl::lib::text::LM'. Ignored.
gcc -DNDEBUG -g -fwrapv -O2 -Wall -Wdate-time -D_FORTIFY_SOURCE=2 -fPIC -I.. -I../kenlm -Ithird_party/openfst-1.6.7/src/include -Ithird_party/ThreadPool -Ithird_party/object_pool -Ithird_party/flashlight -I/usr/local/lib/python3.6/dist-packages/numpy/core/include -I/usr/include/python3.6m -c swigwrapper_wrap.cpp -o temp_build/temp_build/swigwrapper_wrap.o -fPIC -DKENLM_MAX_ORDER=6 -std=c++14 -Wno-unused-local-typedefs -Wno-sign-compare -O3 -DNDEBUG
creating temp_build/temp_build/coqui_stt_ctcdecoder
c++ -pthread -shared -Wl,-O1 -Wl,-Bsymbolic-functions -Wl,-Bsymbolic-functions -Wl,-z,relro -Wl,--no-as-needed -Wdate-time -D_FORTIFY_SOURCE=2 temp_build/temp_build/swigwrapper_wrap.o -o temp_build/temp_build/coqui_stt_ctcdecoder/_swigwrapper.cpython-36m-x86_64-linux-gnu.so first_party.a third_party.a
/usr/local/lib/python3.6/dist-packages/setuptools/dist.py:726: UserWarning: Usage of dash-separated 'build-lib' will not be supported in future versions. Please use the underscore name 'build_lib' instead
  % (opt, underscore_opt)
/usr/local/lib/python3.6/dist-packages/setuptools/dist.py:726: UserWarning: Usage of dash-separated 'build-temp' will not be supported in future versions. Please use the underscore name 'build_temp' instead
  % (opt, underscore_opt)
/usr/local/lib/python3.6/dist-packages/setuptools/dist.py:726: UserWarning: Usage of dash-separated 'build-dir' will not be supported in future versions. Please use the underscore name 'build_dir' instead
  % (opt, underscore_opt)
find temp_build -type f -name "*.o" -delete
DISTUTILS_USE_SDK=1 AS=as CC=gcc CXX=c++ LD=ld LIBEXE= CFLAGS="   " LDFLAGS="-Wl,--no-as-needed"   python ./setup.py bdist_wheel 
running bdist_wheel
running build
running build_ext
running build_py
/usr/local/lib/python3.6/dist-packages/setuptools/dist.py:726: UserWarning: Usage of dash-separated 'build-lib' will not be supported in future versions. Please use the underscore name 'build_lib' instead
  % (opt, underscore_opt)
/usr/local/lib/python3.6/dist-packages/setuptools/dist.py:726: UserWarning: Usage of dash-separated 'build-temp' will not be supported in future versions. Please use the underscore name 'build_temp' instead
  % (opt, underscore_opt)
/usr/local/lib/python3.6/dist-packages/setuptools/dist.py:726: UserWarning: Usage of dash-separated 'build-dir' will not be supported in future versions. Please use the underscore name 'build_dir' instead
  % (opt, underscore_opt)
file coqui_stt_ctcdecoder.py (for module coqui_stt_ctcdecoder) not found
copying ./__init__.py -> temp_build/temp_build/coqui_stt_ctcdecoder
copying ./swigwrapper.py -> temp_build/temp_build/coqui_stt_ctcdecoder
file coqui_stt_ctcdecoder.py (for module coqui_stt_ctcdecoder) not found
installing to temp_build/temp_build
running install
running install_lib
running install_egg_info
running egg_info
creating coqui_stt_ctcdecoder.egg-info
writing coqui_stt_ctcdecoder.egg-info/PKG-INFO
writing dependency_links to coqui_stt_ctcdecoder.egg-info/dependency_links.txt
writing requirements to coqui_stt_ctcdecoder.egg-info/requires.txt
writing top-level names to coqui_stt_ctcdecoder.egg-info/top_level.txt
writing manifest file 'coqui_stt_ctcdecoder.egg-info/SOURCES.txt'
/usr/local/lib/python3.6/dist-packages/setuptools/command/install.py:37: SetuptoolsDeprecationWarning: setup.py install is deprecated. Use build and pip and other standards-based tools.
  setuptools.SetuptoolsDeprecationWarning,
file coqui_stt_ctcdecoder.py (for module coqui_stt_ctcdecoder) not found
reading manifest file 'coqui_stt_ctcdecoder.egg-info/SOURCES.txt'
adding license file 'LICENSE.paddlepaddle'
adding license file 'LICENSE.parlance'
adding license file 'COPYING'
writing manifest file 'coqui_stt_ctcdecoder.egg-info/SOURCES.txt'
Copying coqui_stt_ctcdecoder.egg-info to temp_build/temp_build/coqui_stt_ctcdecoder-1.3.0-py3.6.egg-info
running install_scripts
adding license file "LICENSE.paddlepaddle" (matched pattern "LICEN[CS]E*")
adding license file "LICENSE.parlance" (matched pattern "LICEN[CS]E*")
adding license file "COPYING" (matched pattern "COPYING*")
creating temp_build/temp_build/coqui_stt_ctcdecoder-1.3.0.dist-info/WHEEL
creating 'dist/coqui_stt_ctcdecoder-1.3.0-cp36-cp36m-linux_x86_64.whl' and adding 'temp_build/temp_build' to it
adding 'coqui_stt_ctcdecoder/__init__.py'
adding 'coqui_stt_ctcdecoder/_swigwrapper.cpython-36m-x86_64-linux-gnu.so'
adding 'coqui_stt_ctcdecoder/swigwrapper.py'
adding 'coqui_stt_ctcdecoder-1.3.0.dist-info/COPYING'
adding 'coqui_stt_ctcdecoder-1.3.0.dist-info/LICENSE.paddlepaddle'
adding 'coqui_stt_ctcdecoder-1.3.0.dist-info/LICENSE.parlance'
adding 'coqui_stt_ctcdecoder-1.3.0.dist-info/METADATA'
adding 'coqui_stt_ctcdecoder-1.3.0.dist-info/WHEEL'
adding 'coqui_stt_ctcdecoder-1.3.0.dist-info/top_level.txt'
adding 'coqui_stt_ctcdecoder-1.3.0.dist-info/RECORD'
removing temp_build/temp_build
rm -rf temp_build
Removing intermediate container 09a24518a7f1
 ---> 9eee082d8377
Step 17/27 : RUN pip3 install --upgrade native_client/ctcdecode/dist/*.whl
 ---> Running in 469f67eac7d0
Processing ./native_client/ctcdecode/dist/coqui_stt_ctcdecoder-1.3.0-cp36-cp36m-linux_x86_64.whl
Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from coqui-stt-ctcdecoder==1.3.0) (1.17.3)
Installing collected packages: coqui-stt-ctcdecoder
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Successfully installed coqui-stt-ctcdecoder-1.3.0
Removing intermediate container 469f67eac7d0
 ---> a9a860cf3c61
Step 18/27 : COPY setup.py /code/setup.py
 ---> 2cca430a7e95
Step 19/27 : COPY VERSION /code/VERSION
 ---> 3868a0d52bc2
Step 20/27 : COPY training /code/training
 ---> 54f0e33ec518
Step 21/27 : RUN mkdir -p /code/kenlm/build/
 ---> Running in 9d9c25bfc59c
Removing intermediate container 9d9c25bfc59c
 ---> df3d431ef45e
Step 22/27 : COPY --from=kenlm-build /code/kenlm/build/bin /code/kenlm/build/bin
 ---> 3d48e6f0137c
Step 23/27 : RUN curl -L https://github.com/coqui-ai/STT/releases/download/v0.9.3/convert_graphdef_memmapped_format.linux.amd64.zip | funzip > convert_graphdef_memmapped_format &&     chmod +x convert_graphdef_memmapped_format
 ---> Running in f8acb9f234cd
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   688  100   688    0     0   7166      0 --:--:-- --:--:-- --:--:--  7166
100 22.6M  100 22.6M    0     0  22.0M      0  0:00:01  0:00:01 --:--:-- 13.6M
Removing intermediate container f8acb9f234cd
 ---> 5f55950f00e5
Step 24/27 : RUN LATEST_STABLE_RELEASE=$(curl "https://api.github.com/repos/coqui-ai/STT/releases/latest" | python -c 'import sys; import json; print(json.load(sys.stdin)["tag_name"])')  bash -c 'curl -L https://github.com/coqui-ai/STT/releases/download/${LATEST_STABLE_RELEASE}/native_client.tflite.Linux.tar.xz | tar -xJvf -'
 ---> Running in 27a2028f6cb5
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100 80752    0 80752    0     0   555k      0 --:--:-- --:--:-- --:--:--  555k
  % Total    % Received % Xferd  Average Speed   Time    Time     Time  Current
                                 Dload  Upload   Total   Spent    Left  Speed
100   672  100   672    0     0   8400      0 --:--:-- --:--:-- --:--:--  8400
  0 3143k    0     0    0     0      0      0 --:--:-- --:--:-- --:--:--     0libstt.so
libkenlm.so
libsox.so.3
generate_scorer_package
100 3143k  100 3143k    0     0  7557k      0 --:--:-- --:--:-- --:--:-- 13.1M
LICENSE
stt
coqui-stt.h
KenLM_License_Info.txt
Removing intermediate container 27a2028f6cb5
 ---> 9f29bca7e90e
Step 25/27 : RUN DS_NODECODER=y DS_NOTENSORFLOW=y pip3 install --upgrade -e .
 ---> Running in 23fff3ef06aa
Obtaining file:///code
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting attrdict
  Downloading attrdict-2.0.1-py2.py3-none-any.whl (9.9 kB)
Collecting bs4
  Downloading bs4-0.0.1.tar.gz (1.1 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Collecting coqpit
  Downloading coqpit-0.0.15-py3-none-any.whl (13 kB)
Requirement already satisfied: numpy in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (1.17.3)
Collecting optuna
  Downloading optuna-2.10.0-py3-none-any.whl (308 kB)
Requirement already satisfied: numba<=0.53.1 in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (0.46.0)
Collecting opuslib==2.0.0
  Downloading opuslib-2.0.0.tar.gz (7.3 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: pandas in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (0.25.3)
Collecting progressbar2
  Downloading progressbar2-3.55.0-py2.py3-none-any.whl (26 kB)
Collecting pyogg>=0.6.14a1
  Downloading PyOgg-0.6.14a1.tar.gz (35 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: resampy>=0.2.2 in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (0.2.2)
Requirement already satisfied: requests in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (2.23.0)
Collecting semver
  Downloading semver-2.13.0-py2.py3-none-any.whl (12 kB)
Requirement already satisfied: six in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (1.13.0)
Collecting sox
  Downloading sox-1.4.1-py2.py3-none-any.whl (39 kB)
Requirement already satisfied: soundfile in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (0.10.3.post1)
Requirement already satisfied: tqdm in /usr/local/lib/python3.6/dist-packages (from coqui-stt-training==1.3.0) (4.46.1)
Collecting webdataset
  Downloading webdataset-0.2.4-py3-none-any.whl (46 kB)
Collecting miniaudio
  Downloading miniaudio-1.46-cp36-cp36m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (551 kB)
Requirement already satisfied: llvmlite>=0.30.0dev0 in /usr/local/lib/python3.6/dist-packages (from numba<=0.53.1->coqui-stt-training==1.3.0) (0.30.0)
Requirement already satisfied: scipy>=0.13 in /usr/local/lib/python3.6/dist-packages (from resampy>=0.2.2->coqui-stt-training==1.3.0) (1.3.1)
Collecting beautifulsoup4
  Downloading beautifulsoup4-4.10.0-py3-none-any.whl (97 kB)
Collecting dataclasses
  Downloading dataclasses-0.8-py3-none-any.whl (19 kB)
Requirement already satisfied: cffi>=1.12.0 in /usr/local/lib/python3.6/dist-packages (from miniaudio->coqui-stt-training==1.3.0) (1.14.0)
Collecting sqlalchemy>=1.1.0
  Downloading SQLAlchemy-1.4.32-cp36-cp36m-manylinux_2_5_x86_64.manylinux1_x86_64.manylinux_2_17_x86_64.manylinux2014_x86_64.whl (1.6 MB)
Requirement already satisfied: packaging>=20.0 in /usr/local/lib/python3.6/dist-packages (from optuna->coqui-stt-training==1.3.0) (20.4)
Requirement already satisfied: PyYAML in /usr/local/lib/python3.6/dist-packages (from optuna->coqui-stt-training==1.3.0) (5.3.1)
Collecting cmaes>=0.8.2
  Downloading cmaes-0.8.2-py3-none-any.whl (15 kB)
Collecting colorlog
  Downloading colorlog-6.6.0-py2.py3-none-any.whl (11 kB)
Collecting cliff
  Downloading cliff-3.10.1-py3-none-any.whl (81 kB)
Collecting alembic
  Downloading alembic-1.7.7-py3-none-any.whl (210 kB)
Requirement already satisfied: pytz>=2017.2 in /usr/local/lib/python3.6/dist-packages (from pandas->coqui-stt-training==1.3.0) (2020.1)
Requirement already satisfied: python-dateutil>=2.6.1 in /usr/local/lib/python3.6/dist-packages (from pandas->coqui-stt-training==1.3.0) (2.8.1)
Collecting python-utils>=2.3.0
  Downloading python_utils-3.1.0-py2.py3-none-any.whl (19 kB)
Requirement already satisfied: chardet<4,>=3.0.2 in /usr/local/lib/python3.6/dist-packages (from requests->coqui-stt-training==1.3.0) (3.0.4)
Requirement already satisfied: idna<3,>=2.5 in /usr/local/lib/python3.6/dist-packages (from requests->coqui-stt-training==1.3.0) (2.9)
Requirement already satisfied: certifi>=2017.4.17 in /usr/local/lib/python3.6/dist-packages (from requests->coqui-stt-training==1.3.0) (2020.4.5.2)
Requirement already satisfied: urllib3!=1.25.0,!=1.25.1,<1.26,>=1.21.1 in /usr/local/lib/python3.6/dist-packages (from requests->coqui-stt-training==1.3.0) (1.25.9)
Collecting braceexpand
  Downloading braceexpand-0.1.7-py2.py3-none-any.whl (5.9 kB)
Requirement already satisfied: pycparser in /usr/local/lib/python3.6/dist-packages (from cffi>=1.12.0->miniaudio->coqui-stt-training==1.3.0) (2.20)
Requirement already satisfied: pyparsing>=2.0.2 in /usr/local/lib/python3.6/dist-packages (from packaging>=20.0->optuna->coqui-stt-training==1.3.0) (2.4.7)
Collecting greenlet!=0.4.17
  Downloading greenlet-1.1.2-cp36-cp36m-manylinux_2_17_x86_64.manylinux2014_x86_64.whl (147 kB)
Requirement already satisfied: importlib-metadata in /usr/local/lib/python3.6/dist-packages (from sqlalchemy>=1.1.0->optuna->coqui-stt-training==1.3.0) (1.6.1)
Collecting Mako
  Downloading Mako-1.1.6-py2.py3-none-any.whl (75 kB)
Collecting importlib-resources
  Downloading importlib_resources-5.4.0-py3-none-any.whl (28 kB)
Collecting soupsieve>1.2
  Downloading soupsieve-2.3.1-py3-none-any.whl (37 kB)
Collecting cmd2>=1.0.0
  Downloading cmd2-2.4.0-py3-none-any.whl (150 kB)
Collecting autopage>=0.4.0
  Downloading autopage-0.5.0-py3-none-any.whl (29 kB)
Collecting stevedore>=2.0.1
  Downloading stevedore-3.5.0-py3-none-any.whl (49 kB)
Collecting pbr!=2.1.0,>=2.0.0
  Downloading pbr-5.8.1-py2.py3-none-any.whl (113 kB)
Collecting PrettyTable>=0.7.2
  Downloading prettytable-2.5.0-py3-none-any.whl (24 kB)
Requirement already satisfied: wcwidth>=0.1.7 in /usr/local/lib/python3.6/dist-packages (from cmd2>=1.0.0->cliff->optuna->coqui-stt-training==1.3.0) (0.2.4)
Collecting pyperclip>=1.6
  Downloading pyperclip-1.8.2.tar.gz (20 kB)
  Preparing metadata (setup.py): started
  Preparing metadata (setup.py): finished with status 'done'
Requirement already satisfied: attrs>=16.3.0 in /usr/local/lib/python3.6/dist-packages (from cmd2>=1.0.0->cliff->optuna->coqui-stt-training==1.3.0) (19.3.0)
Collecting typing-extensions
  Downloading typing_extensions-4.1.1-py3-none-any.whl (26 kB)
Requirement already satisfied: zipp>=0.5 in /usr/local/lib/python3.6/dist-packages (from importlib-metadata->sqlalchemy>=1.1.0->optuna->coqui-stt-training==1.3.0) (3.1.0)
Collecting importlib-metadata
  Downloading importlib_metadata-4.8.3-py3-none-any.whl (17 kB)
Requirement already satisfied: MarkupSafe>=0.9.2 in /usr/local/lib/python3.6/dist-packages (from Mako->alembic->optuna->coqui-stt-training==1.3.0) (1.1.1)
Building wheels for collected packages: opuslib, pyogg, bs4, pyperclip
  Building wheel for opuslib (setup.py): started
  Building wheel for opuslib (setup.py): finished with status 'done'
  Created wheel for opuslib: filename=opuslib-2.0.0-py3-none-any.whl size=11009 sha256=e71b6929d3acb133660a10b0095defbbbf20565532032002b98074949e9465d0
  Stored in directory: /root/.cache/pip/wheels/6c/01/88/37797e9e9d157a33eefed22a46aa0bf5044effcec6a9181e41
  Building wheel for pyogg (setup.py): started
  Building wheel for pyogg (setup.py): finished with status 'done'
  Created wheel for pyogg: filename=PyOgg-0.6.14a1-py2.py3-none-any.whl size=35330 sha256=a6ef2f3345e439165e17dc80478a95432542fdf2eca02cad48777783c4dcb4bc
  Stored in directory: /root/.cache/pip/wheels/88/6c/f4/64ad68f9c45076422da74239175b8d66b7495c6f3339c0fc91
  Building wheel for bs4 (setup.py): started
  Building wheel for bs4 (setup.py): finished with status 'done'
  Created wheel for bs4: filename=bs4-0.0.1-py3-none-any.whl size=1272 sha256=d35d4518109740ce5d4cb8a6ad2063405e7e3c74ade75d488b3fbbab72a24d4b
  Stored in directory: /root/.cache/pip/wheels/19/f5/6d/a97dd4f22376d4472d5f4c76c7646876052ff3166b3cf71050
  Building wheel for pyperclip (setup.py): started
  Building wheel for pyperclip (setup.py): finished with status 'done'
  Created wheel for pyperclip: filename=pyperclip-1.8.2-py3-none-any.whl size=11137 sha256=98179a4cfbc956264b71b73736396773d4728f345a9d79612c128d24c43a305d
  Stored in directory: /root/.cache/pip/wheels/95/38/95/e30a7f0b44cb90642de3469f211a3218f93f871789b4f4b46c
Successfully built opuslib pyogg bs4 pyperclip
Installing collected packages: typing-extensions, pyperclip, pbr, importlib-metadata, greenlet, stevedore, sqlalchemy, soupsieve, PrettyTable, Mako, importlib-resources, cmd2, autopage, python-utils, dataclasses, colorlog, cmaes, cliff, braceexpand, beautifulsoup4, alembic, webdataset, sox, semver, pyogg, progressbar2, opuslib, optuna, miniaudio, coqpit, bs4, attrdict, coqui-stt-training
  Attempting uninstall: importlib-metadata
    Found existing installation: importlib-metadata 1.6.1
    Uninstalling importlib-metadata-1.6.1:
      Successfully uninstalled importlib-metadata-1.6.1
  Running setup.py develop for coqui-stt-training
WARNING: Running pip as the 'root' user can result in broken permissions and conflicting behaviour with the system package manager. It is recommended to use a virtual environment instead: https://pip.pypa.io/warnings/venv
Successfully installed Mako-1.1.6 PrettyTable-2.5.0 alembic-1.7.7 attrdict-2.0.1 autopage-0.5.0 beautifulsoup4-4.10.0 braceexpand-0.1.7 bs4-0.0.1 cliff-3.10.1 cmaes-0.8.2 cmd2-2.4.0 colorlog-6.6.0 coqpit-0.0.15 coqui-stt-training-1.3.0 dataclasses-0.8 greenlet-1.1.2 importlib-metadata-4.8.3 importlib-resources-5.4.0 miniaudio-1.46 optuna-2.10.0 opuslib-2.0.0 pbr-5.8.1 progressbar2-3.55.0 pyogg-0.6.14a1 pyperclip-1.8.2 python-utils-3.1.0 semver-2.13.0 soupsieve-2.3.1 sox-1.4.1 sqlalchemy-1.4.32 stevedore-3.5.0 typing-extensions-4.1.1 webdataset-0.2.4
Removing intermediate container 23fff3ef06aa
 ---> b3a9bc9a9998
Step 26/27 : COPY . /code
 ---> 64ee6415ffd0
Step 27/27 : RUN ./bin/run-ldc93s1.sh && rm -rf ~/.local/share/stt
 ---> Running in ee08beb70f49
+ '[' '!' -f train.py ']'
+ '[' '!' -f data/smoke_test/ldc93s1.csv ']'
+ echo 'Downloading and preprocessing LDC93S1 example data, saving in ./data/smoke_test.'
+ python -u bin/import_ldc93s1.py ./data/smoke_test
Downloading and preprocessing LDC93S1 example data, saving in ./data/smoke_test.
2022-03-23 00:29:26.410714: I tensorflow/stream_executor/platform/default/dso_loader.cc:48] Successfully opened dynamic library libcudart.so.11.0
WARNING:tensorflow:Deprecation warnings have been disabled. Set TF_ENABLE_DEPRECATION_WARNINGS=1 to re-enable them.
Found archive "./data/smoke_test/LDC93S1.wav" - not downloading.
Found archive "./data/smoke_test/LDC93S1.txt" - not downloading.
+ checkpoint_dir=/root/.local/share/stt/ldc93s1
+ export CUDA_VISIBLE_DEVICES=0
+ CUDA_VISIBLE_DEVICES=0
+ python -m coqui_stt_training.train --alphabet_config_path data/alphabet.txt --show_progressbar false --train_files data/smoke_test/ldc93s1.csv --test_files data/smoke_test/ldc93s1.csv --train_batch_size 1 --test_batch_size 1 --n_hidden 100 --epochs 200 --checkpoint_dir /root/.local/share/stt/ldc93s1
WARNING:tensorflow:Deprecation warnings have been disabled. Set TF_ENABLE_DEPRECATION_WARNINGS=1 to re-enable them.
Traceback (most recent call last):
  File "/usr/lib/python3.6/runpy.py", line 193, in _run_module_as_main
    "__main__", mod_spec)
  File "/usr/lib/python3.6/runpy.py", line 85, in _run_code
    exec(code, run_globals)
  File "/code/training/coqui_stt_training/train.py", line 34, in <module>
    from . import evaluate
  File "/code/training/coqui_stt_training/evaluate.py", line 16, in <module>
    from .deepspeech_model import create_model, reset_default_graph
  File "/code/training/coqui_stt_training/deepspeech_model.py", line 25, in <module>
    from .util.config import Config
  File "/code/training/coqui_stt_training/util/config.py", line 16, in <module>
    from .augmentations import NormalizeSampleRate, parse_augmentations
  File "/code/training/coqui_stt_training/util/augmentations.py", line 26, in <module>
    from .sample_collections import samples_from_source, unpack_maybe
  File "/code/training/coqui_stt_training/util/sample_collections.py", line 21, in <module>
    import webdataset as wds
  File "/usr/local/lib/python3.6/dist-packages/webdataset/__init__.py", line 21, in <module>
    from .cache import (
  File "/usr/local/lib/python3.6/dist-packages/webdataset/cache.py", line 43
    while data := stream.read(chunk_size):
                ^
SyntaxError: invalid syntax
The command '/bin/sh -c ./bin/run-ldc93s1.sh && rm -rf ~/.local/share/stt' returned a non-zero code: 1
```

--- a/setup.py
+++ b/setup.py
@@ -30,7 +30,7 @@ def main():
         "sox",
         "soundfile",
         "tqdm",
-        "webdataset",
+        "webdataset==0.1.103",
         "miniaudio",
     ]
 

